handlers/config.go

```diff
@@ -51,6 +51,7 @@ type DeploymentConfig struct {
 type PublicConfig struct {
 	AccessibleNamespaces []string                    `json:"accesibleNamespaces,omitempty"`
 	AuthStrategy         string                      `json:"authStrategy,omitempty"`
+	AmbientProfile       bool                        `json:"ambientProfile,omitempty"`
```

Comment:
another "profile" (see my earlier comments about this word and my alternatives).